system-probe_arm64/Dockerfile

```diff
@@ -37,4 +42,10 @@ RUN mkdir -p $GOPATH/src/github.com/DataDog/datadog-agent
 RUN wget "https://github.com/llvm/llvm-project/releases/download/llvmorg-11.0.0/clang+llvm-11.0.0-aarch64-linux-gnu.tar.xz" -O /tmp/clang.tar.xz  -o /dev/null
 RUN mkdir -p /opt/clang
 RUN tar xf /tmp/clang.tar.xz  -C /opt/clang --strip-components=1
-ENV PATH /opt/clang/bin:${PATH}
+
+RUN python3 -m pip install invoke==1.4.1
```

Comment:
Most build steps do not actually do `python3 -m pip install requirements.txt`, they rely on the fact that `invoke` is already here in the image (you can check the other Linux build images in this repo, they all install invoke).

You can also see that this is actually not a change: `invoke` was already installed in this image, I only changed the way it gets installed (ie. not with `apt` because the packaged version is old on buster).

The question (which won't be answered in this PR) is: right now, our build images and gitlab jobs are inconsistent. `invoke` is sometimes already installed in the image (but sometimes it's not). Moreover, some jobs install the python requirements, others do not.

Most jobs do not actually need the python requirements: they only need `invoke`. The jobs that really need python requirements are unit tests and some (really precise) invoke tasks that need some more python packages.

There are multiple solutions to this:
- include invoke in all images, let gitlab jobs install other requirements if needed
- do not include invoke, make all jobs run the python requirements (even if most of them aren't used by the job - or maybe we should split requirements?)

Reply:
Makes sense, I hadn't seen that it was already installed! It's inconsistent, yes, but it's a bit difficult to install only the requirements we need on each place and it's not something that is much of an issue so we don't need to deal with it now IMO